backend/tests/unit_tests/config/payloads/payloads.json

```diff
@@ -170,47 +237,46 @@
                   "type": "string"
                 }
               },
-              "required": [
-                "name",
-                "date"
-              ]
+              "required": ["name", "date"]
             }
-          }],
-          "rows_affected": 0
-        },
-        "return_value": {
-          "rows_affected": 0,
-          "response": [{"data": {"name": "test_list_w_cd_key", "date": "2018-01-01"}}]
-        }
-      },
-      "test_list_o_cd_key": {
-        "config_definition_key": "",
-        "data": {
-          "page": 1,
-          "page_size": 10
-        }
-      },
-      "test_list_n_page": {
-        "config_definition_key": "sample_config_definition",
-        "data": {
-          "page": -1,
-          "page_size": 10
-        }
-      },
-      "test_list_n_nlimit": {
-        "config_definition_key": "sample_config_definition",
-        "data": {
-          "page": 1,
-          "page_size": -1
-        }
+          }
+        ],
+        "rows_affected": 0
       },
-      "test_list_n_plimit": {
-        "config_definition_key": "sample_config_definition",
-        "data": {
-          "page": 1,
-          "page_size": 101
-        }
+      "return_value": {
+        "rows_affected": 0,
+        "response": [
+          { "data": { "name": "test_list_w_cd_key", "date": "2018-01-01" } }
+        ]
+      }
+    },
+    "test_list_o_cd_key": {
+      "config_definition_key": "",
+      "data": {
+        "page": 1,
+        "page_size": 10
+      }
+    },
+    "test_list_n_page": {
+      "config_definition_key": "sample_config_definition",
+      "data": {
+        "page": -1,
+        "page_size": 10
+      }
+    },
+    "test_list_n_nlimit": {
+      "config_definition_key": "sample_config_definition",
+      "data": {
+        "page": 1,
+        "page_size": -1
+      }
+    },
+    "test_list_n_plimit": {
+      "config_definition_key": "sample_config_definition",
+      "data": {
+        "page": 1,
+        "page_size": 101
       }
```

Comment:
I guess this could be done by using some new assertions like

```
if expect_error:
    with pytest.raises(ValueError) as exc_info:
        u_config_definition(config_key, updated_indexes_list)
    assert str(exc_info.value) == "Invalid request: page_size must be less than or equal to 100"
    mock_execute_query.assert_not_called()
    return
```

Can you make a new issue for this? This can be called `tests(general): adding ValueError message assertions`?